modules/extensions-support/src/main/java/org/mule/runtime/module/extension/internal/loader/enricher/ParameterAllowedStereotypesDeclarionEnricher.java

```diff
@@ -83,69 +64,39 @@ void enrich(ExtensionDeclaration extension) {
         @Override
         protected void onParameter(ParameterGroupDeclaration parameterGroup, ParameterDeclaration declaration) {
           declaration.getModelProperty(ImplementingParameterModelProperty.class)
-              .ifPresent(param -> declaration.setAllowedStereotypeModels(getStereotypes(param.getParameter(), typesByClassName)));
+              .ifPresent(param -> declaration.getAllowedStereotypeModels()
+                  .addAll(getStereotypes(param.getParameter(), typesByClassName)));
           declaration.getModelProperty(DeclaringMemberModelProperty.class)
-              .ifPresent(field -> declaration
-                  .setAllowedStereotypeModels(getStereotypes(field.getDeclaringField(), typesByClassName)));
+              .ifPresent(field -> declaration.getAllowedStereotypeModels()
+                  .addAll(getStereotypes(field.getDeclaringField(), typesByClassName)));
         }
       }.walk(extension);
     }
 
     private List<StereotypeModel> getStereotypes(AnnotatedElement element, Map<String, ObjectType> typesByClassName) {
-      ConfigReferences references = element.getAnnotation(ConfigReferences.class);
-      if (references != null) {
-        return stream(references.value()).map(ref -> newStereotype(ref.name(), ref.namespace())
-            .withParent(CONFIG)
-            .build())
-            .collect(toList());
-      }
-
-      ConfigReference ref = element.getAnnotation(ConfigReference.class);
-      if (ref != null) {
-        return singletonList(newStereotype(ref.name(), ref.namespace()).withParent(CONFIG).build());
-      }
-
-      if (element.getAnnotation(FlowReference.class) != null) {
-        return singletonList(FLOW);
```

Comment:
why was this removed / where is this logic now?

Reply:
this is now in the model parsers